<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientCrawler</class>
 <widget class="QWidget" name="ClientCrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientCrawler</string>
  </property>
  <widget class="QLabel" name="labelAdresseServeur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse du serveur</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumeroPort">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAdresseServeur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnexion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDistance">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DIstance</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPortServeur">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberDistance">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInformations">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>580</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Informations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeft">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Fleches.qrc">
     <normaloff>:/img/images/arrow_left.png</normaloff>:/img/images/arrow_left.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRight">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Fleches.qrc">
     <normaloff>:/img/images/arrow_right.png</normaloff>:/img/images/arrow_right.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUp">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Fleches.qrc">
     <normaloff>:/img/images/arrow_up.png</normaloff>:/img/images/arrow_up.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDown">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Fleches.qrc">
     <normaloff>:/img/images/arrow_down.png</normaloff>:/img/images/arrow_down.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Fleches.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ClientCrawler</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
